R/parsed.R

```diff
@@ -9,6 +9,18 @@ compute_parse_data_flat_with_ws <- function(text) {
   parse_data_with_ws
 }
 
+#' Pre-processing parse data
+#'
+#' @details Preprocessing includes
+#' * removing non-terminal entries.
+#' * remove columns id, parent and terminal.
+#' * adding a start token.
+#' * add linebreak and space information.
+#' * replace spaces in comments at the end of the line.
```

Comment:
well it should probably read "remove". Should I change that? I thought it is in comments only since relevant line is #70
`mutate_(text = ~if_else(token == "COMMENT", gsub(" +$", "", text), text))`